forem/forem
# Unified Embed Documentation

## What type of PR is this? (check all applicable)


## Description
In light of our new [Unified Embeds](https://github.com/forem/forem/issues/15099) for Liquid Tags, this PR updates the related documentation. 

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings
Check the Liquid Tag documentation in the following locations:
- sidebar of the Article Editor
- Liquid Tag modal with the Editor view (`/new#liquid`)
- The Editor Guide (`/p/editor_guide`)

### UI accessibility concerns?
Will address any issues uncovered

## Added/updated tests?

- [X] No, and this is why: _documentation updates only_

## [Forem core team only] How will this change be communicated?

- [X] I will share this change in a [Changelog](https://forem.dev/t/changelog) and a [dev.to](http://dev.to) post

Forem.dev draft post [HERE](https://forem.dev/foremteam/changelog-new-unified-way-to-embed-liquid-tags-111o-temp-slug-7381974?preview=1595d8714b6739787b145a67fbadd5aded6fd99af6fd5d095a0e441684f55fbaef0a5b76831595f50ebcb6bfe2c642e55ee05812f15011c9a37e2ba6)
Dev.to post HERE

## [optional] Are there any post deployment tasks we need to perform?
None

## [optional] What gif best describes this PR or how it makes you feel?

![African masquerade dancing](https://media.giphy.com/media/SQ8ffKoedTWrS/giphy-downsized.gif)
